runtime/runtime-params-estimator/src/utils.rs

```diff
@@ -203,6 +203,25 @@ pub(crate) fn fn_cost_with_setup(
 
         let (gas_cost, ext_costs) =
             aggregate_per_block_measurements(block_size, measurements, Some(overhead));
+
+        // flat storage check: we only expect TTN costs for writes
+        #[cfg(feature = "protocol_feature_flat_state")]
+        // TODO(#7327): This assertion is ignored, we know flat storage doesn't
```

Comment:
Actually I'm working on enabling flat storage for runtime-params-estimator as part of #8322, so you might want to update `TODO` here :)

Reply:
done!